daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/scenario/ScenarioLedger.scala

```diff
@@ -83,7 +83,7 @@ object ScenarioLedger {
       globalImplicitDisclosure: Relation[ContractId, Party],
       failedAuthorizations: FailedAuthorizations,
   ) {
-    def disclosures(coidToEventId: ContractId => EventId): Relation[EventId, Party] =
+    def disclosures(coidToEventId: ContractId => EventId): Map[EventId, Set[Party]] =
```

Comment:
While I’m not really a fan of synonyms it also seems a bit confusing to work with `Relation.…` in the body of this method but then not use the type synonym.

Reply:
Good catch. That's a relic from a failed attempt to change some things. I'll bring it back to its original state. I should have self-reviewed my PR better. 😞